.github/workflows/test.yml

```diff
@@ -28,45 +28,36 @@ jobs:
   test:
     name: unittests
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
-        toxenv:
-            - py36-dj22-wag27
-            - py36-dj22-waglatest
-            - py39-dj22-waglatest
-            - py39-dj32-waglatest
-            - py39-dj40-waglatest
-        include:
-          - toxenv: py36-dj22-wag27
-            python-version: 3.6
-          - toxenv: py36-dj22-waglatest
-            python-version: 3.6
-          - toxenv: py39-dj22-waglatest
-            python-version: 3.9
-          - toxenv: py39-dj32-waglatest
-            python-version: 3.9
-          - toxenv: py39-dj40-waglatest
-            python-version: 3.9
+        python: ['3.7', '3.8', '3.9', '3.10']
+        django: ['3.1', '3.2']
+        wagtail: ['2.15']
+        include: 
+          - python: '3.9'
+            django: '3.2'
+            wagtail: '2.16'
+          - python: '3.10'
+            django: '3.2'
+            wagtail: '3.0'
+          - python: '3.10'
+            django: '4.0'
+            wagtail: '3.0'
```

Comment:
This set of changes (and to our tox setup below) give me pause. We try to keep our CFPB Python/Django/Wagtail libraries consistent with each other in their testing/CI setup to make maintenance that little bit easier for us (we're a small team, and we want to make the effort of maintaining these as separate libraries as simple as possible).

This is not a bad set of changes, but they're inconsistent with the other libraries we maintain. Changes like these should probably happen separately to a PR like this.